### PR TITLE
Customization of puppeteer launch arguments through config.json

### PR DESCRIPTION
**Pull request by japboy**
I'm running Rendertron on my local development environment and my server is self-signed https server. This cause `ERR_CERT_AUTHORITY_INVALID` error if I try to access to something like;

```
http://localhost:3000/render/https://192.168.10.2:9000/foo/bar

# cause an error below:
# Error: net::ERR_CERT_AUTHORITY_INVALID at https://192.168.10.2:9000/foo/bar
```

so I want to pass `--ignore-certificate-errors` argument to puppeteer. And I made it possible by passing `args` options through `config.json` like this below;

```
{
 "args": [
   "--ignore-certificate-errors"
 ]
}
```

It would be great if I could add necessary arguments through the `config.json`

### CODE REVIEW

**Comment by japboy** [comment]
I signed it!

File: src/rendertron.ts [hunk 1]
```diff
@@ -8,13 +8,15 @@ import * as koaLogger from 'koa-logger';
 import * as path from 'path';
 import * as puppeteer from 'puppeteer';
 import * as url from 'url';
+import { isArray, union } from 'lodash';
```
Comment (AVGP) [inline src/rendertron.ts]: Can we avoid dragging in this dependency for just these two simple features?

`isArray` -> `Array.isArray`
`union` -> `[...new Set([...a, ...b])];`

Comment (michie1) [reply src/rendertron.ts]: @japboy Can I create a (new) PR to improve this? Or do you need to have some time and want to do it yourself?

Comment (vzaidman) [reply src/rendertron.ts]: @japboy i'd be happy to help too

Comment (AVGP) [reply src/rendertron.ts:11]: Thanks for your patience, I will merge this PR and make the necessary changes.

**Pull request closed without merging**